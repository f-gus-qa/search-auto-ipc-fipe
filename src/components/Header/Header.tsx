import { Box, Typography } from '@material-ui/core';

function Header() {
  return (
    <Box mb={2} textAlign="center">
      <Box mb={1}>
        <Typography variant="h4">
          Tabela Fipe
        </Typography>
      </Box>
      <Typography variant="h6">
        Consulte o valor de um ve√≠culo de forma gratuita
      </Typography>
    </Box>
  );
}

export default Header;
