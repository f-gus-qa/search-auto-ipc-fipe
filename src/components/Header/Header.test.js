import { create } from 'react-test-renderer';
import React from 'react';
import Header from '.';

describe('<Header>', () => {
  it('check renders structure', () => {
    const { toJSON } = create(<Header />);

    expect(toJSON()).toMatchSnapshot();
  });
});
