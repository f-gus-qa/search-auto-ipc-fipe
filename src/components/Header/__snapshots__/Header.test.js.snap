// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> check renders structure 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiBox-root MuiBox-root-2"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Tabela Fipe
    </h4>
  </div>
  <h6
    className="MuiTypography-root MuiTypography-h6"
  >
    Consulte o valor de um ve√≠culo de forma gratuita
  </h6>
</div>
`;
